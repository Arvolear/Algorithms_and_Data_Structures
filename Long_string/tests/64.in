9881407387223922871
429766871172173582889023247048602678467775002