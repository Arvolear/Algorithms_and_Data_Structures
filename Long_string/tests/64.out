429766871172173582889023256930010065691697873
429766871172173582889023237167195291243852131
4246701535584828022459411358591655014904323134517533833429870742
43492475750755586457435081